src/features/Integrations/renderIntegrationFormField.tsx

```diff
@@ -0,0 +1,134 @@
+import {
+  CustomBoolField,
+  CustomDescriptionField,
+  CustomInputField,
+  CustomLabel,
+} from "../../components/IntegrationsView/CustomFieldsAndWidgets";
+import type {
+  Integration,
+  IntegrationField,
+  IntegrationPrimitive,
+} from "../../services/refact";
+import { DataList, Flex } from "@radix-ui/themes";
+import { toPascalCase } from "../../utils/toPascalCase";
+import { SmartLink } from "../../components/SmartLink";
+
+type FieldType = "string" | "bool" | "int";
+
+const isFieldType = (value: string): value is FieldType => {
+  return ["string", "bool", "int"].includes(value);
+};
+
+const getDefaultValue = ({
+  field,
+  values,
+  fieldKey,
+  f_type,
+}: {
+  fieldKey: string;
+  values: Integration["integr_values"];
+  field: IntegrationField<NonNullable<IntegrationPrimitive>>;
+  f_type: "bool" | "int" | "string";
+}) => {
+  if (values[fieldKey]) {
```

Comment:
Could be a falsy value `0, null ""` ?
```
if(fieldKey in values)  {

}
```
or 
```
if(values[fieldKey] !== undefined) {}
```

Reply:
Good point 👍🏻 
Done, pushed